spring-cloud-dataflow-docs/src/main/asciidoc/configuration.adoc

```diff
@@ -1083,113 +1083,70 @@ being set to `true`.
 
 [[configuration-monitoring-management]]
 == Monitoring and Management
-The Spring Cloud Data Flow server is a Spring Boot application that includes the http://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#production-ready[Actuator
+The Spring Cloud Data Flow server is a Spring Boot 1.5 application that includes the  https://docs.spring.io/spring-boot/docs/1.5.12.RELEASE/reference/htmlsingle/#production-ready[Actuator
 library], which adds several production ready features to help you monitor and manage your application.
 
-The Actuator library adds HTTP endpoints under the context path `/management` that is also
-a discovery page for available endpoints.  For example, there is a `health` endpoint
-that shows application health information and an `env` that lists properties from
-Spring's `ConfigurableEnvironment`.  By default, only the health and application info
-endpoints are accessible.  The other endpoints are considered to be sensitive
-and need to be http://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#production-ready-customizing-endpoints[enabled explicitly via configuration].  If you enable
-sensitive endpoints, you should also
-<<configuration-security,secure the Data Flow server's endpoints>> so that
-information is not inadvertently exposed to unauthenticated users.  The local Data Flow server has security disabled by default, so all actuator endpoints are available.
+The Actuator library adds HTTP endpoints under the context path `/management` which is a discovery page for available managerment endpoints.
+For example, there is a `health` endpoint that shows application health information and an `env` that lists properties from Spring's `ConfigurableEnvironment`.
+By default, only the health and application info endpoints are accessible.  The other endpoints are considered to be sensitive
+and need to be  https://docs.spring.io/spring-boot/docs/1.5.12.RELEASE/reference/htmlsingle/#production-ready-customizing-endpoints[enabled explicitly via configuration].
+If you enable sensitive endpoints, you should also <<configuration-security,secure the Data Flow server's endpoints>> so that information is not inadvertently exposed to unauthenticated users.
```

Comment:
The formatting here seems a little odd. "<<configuration-security,secure the Data Flow server's endpoints>>"